package.json

```diff
@@ -28,8 +28,7 @@
     "docs": "npx jsdoc -c ./jsdoc.json",
     "update": "npx tsc",
     "postinstall": "npx concurrently -c \"auto\" --names \"react-dependencies\" \"cd react && npm install\"",
-    "patch:lit-translate": "unzip -o ./scripts/lit-translate.zip -d ./node_modules/lit-translate && cp -f ./scripts/lit-translate-index.js ./node_modules/lit-translate/index.js",
-    "prepare": "husky install"
```

Comment:
You might have deleted this line by mistake. Please check carefully before pushing changes and requesting a review. @gee05053

Reply:
@yomybaby
Sorry.. I will check one more time and request review again